test/automation-bot.ts

```diff
@@ -214,6 +218,7 @@ describe('AutomationBot', async () => {
                 [false],
                 [7],
                 [triggerData],
```

Comment:
we should add a test for trigger replacement in `automationBot` tests - don't know why is it not here. I know it was tested on frontend :)